sfts/s3_to_sfts.py

```diff
@@ -161,19 +225,23 @@ def is_processed():
 try:
     xfer_jar = f"{xfer_path}/xfer.jar"
     jna_jar = f"{xfer_path}/jna.jar"
-    print(("trying to call subprocess:\nxfer.jar: "
+    logger.info(("trying to call subprocess:\nxfer.jar: "
            f"{xfer_jar}\njna.jar : {jna_jar}"))
-    subprocess.check_call(
+    output = subprocess.check_output(
         ["java", "-classpath", f"{xfer_jar}:{jna_jar}",
          "xfer",
          f"-user:{sfts_user}",
          f"-password:{sfts_pass}",
          f"-s:{sfts_conf}",
          "filetransfer.gov.bc.ca"])
     xfer_proc = True
+    logger.info(output)
```

Comment:
If you choose not to use an encoding flag on `subprocess.check_output` above, then you may alternatively decode this here.

Something like: `logger.info(output.decode('utf-8'))` would be a place to start.